theme/templates/resource-landing-page/coverage.html

```diff
@@ -154,7 +154,19 @@ <h3>Coverage</h3>
                         </div>
 
                         <br>
-                        <div class="col-xs-12"><h4>Spatial: </h4></div>
+                        <div class="col-xs-12">
+                            <div class="row">
+                                <div class="col-xs-1">
+                                    <h4>Spatial:</h4>
+                                </div>
+                                <div class="col-xs-11">
```

Comment:
@pkdash , `col-xs-11` will cause the button to overlap the header on mobile devices.
![image](https://user-images.githubusercontent.com/2448568/49552946-cce18900-f8b2-11e8-8c05-6818b974ef0a.png)

It is probably better to render the delete button next to the header. Or consider using CSS flex. 
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
I did not use css but used bootstrap grid columns and tested to work OK with mobile device>

![image](https://user-images.githubusercontent.com/2523434/49625426-799b3380-f9a3-11e8-9f23-3545edf82e58.png)